src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -274,7 +274,9 @@ private String getAddress(Principal principal) {
     @Path("/facets/bundles")
     @Operation(summary = "Return a thin list of configured bundles. This can be used to configure a filter in the UI")
     public Multi<Facet> getBundleFacets(@Context SecurityContext sec) {
```

Comment:
I think this could also be a `Uni<List<Facet>>`

Reply:
The `Uni` is transformed to a `Multi` because we have to transform each item of the `List` individually.

We could transform the `Multi` back to a `Uni` before returning it, or never transform to a `Multi` and transform the `Uni` items using old school Java, or leave it that way. I honestly don't know what's best for performances (because all these solutions are equally maintainable).